src/app/components/project-map/project-map.component.scss

```diff
@@ -382,6 +382,9 @@ g.node text,
 .context-menu-items .mat-menu-item:focus {
   background: none;
 }
+.context-menu-items .mat-menu-item:hover {
```

Comment:
Does this apply only to the light theme?

Reply:
I will try @grossmj but we are using same menu with same class in light and dark both theme.